stable/lamp/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+description: Modular and transparent LAMP stack chart supporting PHP-FPM, Release Cloning, LoadBalancer, Ingress, SSL and lots more!
+name: lamp
+version: 0.1.0
```

Comment:
Add `appVersion`.

Reply:
We can leave this off since there is a conglomeration of app versions.